plugins/source/aws/resources/services/ssoadmin/instances_mock_test.go

```diff
@@ -16,6 +16,7 @@ func buildInstances(t *testing.T, ctrl *gomock.Controller) client.Services {
 	im := types.InstanceMetadata{}
 	ps := types.PermissionSet{}
 	as := types.AccountAssignment{}
+	ip := ""
```

Comment:
I think this will need to be valid JSON if the column type is to be JSON, and faking `&ip` lower down with a string object will probably lead to a random string (not valid JSON). 

Any valid JSON should do for the purposes of this test though, so maybe just instantiate this variable as `{"key": "value}` or similar